tests/purs/passing/RowsInInstanceContext.purs

```diff
@@ -15,9 +14,13 @@ instance refl :: TypeEquals a a where
 
 newtype RecordNewtype = RecordNewtype { x :: String }
 
+class OldStyleNewtype t a where
+  wrap :: a -> t
+  unwrap :: t -> a
+
 instance newtypeRecordNewtype ::
   TypeEquals inner { x :: String }
-    => Newtype RecordNewtype inner where
+    => OldStyleNewtype RecordNewtype inner where
```

Comment:
I'm not exactly sure what this test was testing exactly, but it seems that the `Newtype` class was used only incidentally, as an example of whatever it is that's being tested. So instead of migrating this to the new `Newtype` class, I created another one,

Reply:
Yeah, I think it is unrelated - it's testing that rows in an instance context are allowed / work correctly.